samples/subsys/shell/fs/README.rst

```diff
@@ -6,12 +6,12 @@ File system shell example
 Overview
 ********
 
-This example provides shell access to a NFFS file system partition in flash.
+This example provides shell access to a littlefs file system partition in flash.
```

Comment:
Inconsistent naming of LittleFS within documentation; varies from littlefs to Littlefs to LittleFS